packages/router-plugin/src/router-generator.ts

```diff
@@ -95,5 +98,17 @@ export const unpluginRouterGeneratorFactory: UnpluginFactory<
         await run(generate)
       })
     },
+    async webpack() {
+      userConfig = await getConfig(options, ROOT)
+
+      await run(generate)
+
+      // webpack watcher doesn't register newly created files
+      const routesDirectoryPath = getRoutesDirectoryPath()
+      const chokidar = await import('chokidar')
```

Comment:
Doesn't `watchChange` work?

Asking since the unplugin folks have it listed as being supported for webpack: https://unplugin.unjs.io/guide/

Reply:
Nevermind, I clearly didn't read the comment 😅.

Reply:
Yeah, it’s weird. I tried to register the folder through `this.addWatchFile` in `buildStart` but it didn’t work either. We might look into unplugin itself and fix it there.